clients/include/testing_gemv_batched.hpp

```diff
@@ -68,16 +71,16 @@ hipblasStatus_t testing_gemv_batched(const Arguments& argus)
 
     // arrays of pointers-to-host on host
     host_batch_vector<T> hA(A_size, 1, batch_count);
-    host_batch_vector<T> hx(X_size, incx, batch_count);
-    host_batch_vector<T> hy(Y_size, incy, batch_count);
-    host_batch_vector<T> hy_cpu(Y_size, incy, batch_count);
-    host_batch_vector<T> hy_host(Y_size, incy, batch_count);
-    host_batch_vector<T> hy_device(Y_size, incy, batch_count);
+    host_batch_vector<T> hx(X_size, 1, batch_count);
```

Comment:
This is fine, but `host_batch_vector<T> hx(dim_x, incx, batch_count);` might be more in the spirit of the `host_batch_vector` type. Can be applied to the other vectors here and in the other batched file.